libs/sdk-ui-filters/src/AttributeFilter/Components/DropdownButton/AttributeFilterDropdownButton.tsx

```diff
@@ -88,6 +89,16 @@ export interface IAttributeFilterDropdownButtonProps {
      */
     icon?: ReactNode;
 
+    /**
+     * Customize attribute filter tooltip component.
+     *
+     * @remarks
+     * -  If not provided, the default implementation {@link AttributeFilterDetails} will be used.
+     *
+     * @beta
+     */
+    TooltipComponent?: React.ComponentType<IAttributeFilterDetailsProps>;
```

Comment:
Rename it to TooltipComponent and change interface ... this I guess should be without props. Its just content what will be displayed.